### PR TITLE
Issue #8169: Making regex to validate missed space in comments in non files

### PR DESCRIPTION
**Pull request by HuGanghui**
Issue #8169: Making regex to validate missed space in comments in non main files

### CODE REVIEW

File: config/checkstyle_non_main_files_checks.xml [hunk 1]
```diff
@@ -38,4 +38,19 @@
       <property name="message"
                value="Two or more test cases are required for unit testing in Xpath "/>
     </module>
+    <module name="RegexpSingleline">
+        <property name="format" value="&lt;!--[^ ]"/>
+        <property name="fileExtensions" value="xml, properties"/>
+        <property name="message" value="space is required in comments"/>
```
Comment (strkkk) [inline config/checkstyle_non_main_files_checks.xml:44]: `space is required in comments` -> space is required after comment start

File: config/checkstyle_non_main_files_checks.xml [hunk 2]
```diff
@@ -38,4 +38,19 @@
...
+    </module>
+    <module name="RegexpSingleline">
+        <property name="format" value="[^ ]--&gt;"/>
+        <property name="fileExtensions" value="xml, properties"/>
+        <property name="message" value="space is required in comments"/>
```
Comment (strkkk) [inline config/checkstyle_non_main_files_checks.xml:49]: `space is required in comments` -> space is required before comment end

File: config/checkstyle_non_main_files_checks.xml [hunk 3]
```diff
@@ -38,4 +38,19 @@
...
+    </module>
+    <module name="RegexpSingleline">
+        <property name="format" value=" #[^ !]"/>
+        <property name="fileExtensions" value="sh, yml"/>
+        <property name="message" value="space is required in comments"/>
```
Comment (strkkk) [inline config/checkstyle_non_main_files_checks.xml:54]: same here

File: config/checkstyle_non_main_files_checks.xml (hunk 1 above)
Comment (HuGanghui) [reply config/checkstyle_non_main_files_checks.xml]: done

File: config/checkstyle_non_main_files_checks.xml (hunk 2 above)
Comment (HuGanghui) [reply config/checkstyle_non_main_files_checks.xml]: done

File: config/checkstyle_non_main_files_checks.xml (hunk 3 above)
Comment (HuGanghui) [reply config/checkstyle_non_main_files_checks.xml]: done

File: config/checkstyle_non_main_files_checks.xml [hunk 4]
```diff
@@ -38,4 +38,19 @@
...
+        <property name="fileExtensions" value="xml, properties"/>
+        <property name="message" value="space is required after comment start"/>
+    </module>
+    <module name="RegexpSingleline">
+        <property name="format" value="[^ ]--&gt;"/>
+        <property name="fileExtensions" value="xml, properties"/>
+        <property name="message" value="space is required before comment end"/>
+    </module>
+    <module name="RegexpSingleline">
+        <property name="format" value=" #[^ !]"/>
```
Comment (strkkk) [inline config/checkstyle_non_main_files_checks.xml:52]: ` #[^ !]"` regex value has space before it. Is it necessary?

File: config/checkstyle_non_main_files_checks.xml [hunk 5]
```diff
@@ -38,4 +38,19 @@
...
+        <property name="fileExtensions" value="sh, yml"/>
```
Comment (strkkk) [inline config/checkstyle_non_main_files_checks.xml:53]: I think yaml should be added, just in case

Comment (HuGanghui) [reply config/checkstyle_non_main_files_checks.xml]: done

File: config/checkstyle_non_main_files_checks.xml (hunk 4 above)
Comment (HuGanghui) [reply config/checkstyle_non_main_files_checks.xml:52]: I think it will make the format of comment consistent at least.

**Comment by rnveach** [comment]
@HuGanghui CI is failing.

**Comment by HuGanghui** [comment]
@rnveach the reason that CI is failing is this issue is not approved, please review and approve it #8169

**Comment by HuGanghui** [comment]
@rnveach the reason that CI is failing is that the Issue #8169: Making regex to validate missed space in comments in non main files not approved, if possible, please review and approve~

**Comment by rnveach** [comment]
@HuGanghui Issue was approved. Sorry for the delay. Please rebase over latest master to ensure CI still passes.

**Comment by HuGanghui** [comment]
@rnveach ok, rebase to latest master

**Pull request merged**